flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateChunkReaderTest.java

```diff
@@ -68,27 +81,28 @@ public void testBufferRecycledOnSuccess() throws IOException, InterruptedExcepti
                     .readChunk(stream, serializer.getHeaderLength(), handler, "channelInfo", 0);
         } finally {
             checkState(!handler.requestedBuffers.isEmpty());
-            assertTrue(
-                    handler.requestedBuffers.stream()
-                            .allMatch(TestChannelStateByteBuffer::isRecycled));
+            assertThat(
+                            handler.requestedBuffers.stream()
+                                    .allMatch(TestChannelStateByteBuffer::isRecycled))
+                    .isTrue();
```

Comment:
```suggestion
            assertThat(handler.requestedBuffers.stream())
                    .allMatch(TestChannelStateByteBuffer::isRecycled);
```
assertJ allows applying conditions to streams